examples/zookeeper/template.json

```diff
@@ -0,0 +1,265 @@
+{
+  "kind": "Template",
+  "apiVersion": "v1beta1",
+  "labels": {
+    "template": "zookeeper"
+  },
+  "metadata": {
+    "annotations": {
+      "description": "Example of replicated Zookeeper 3.4.6 server",
+      "tags": "database,zookeeper",
+      "iconClass" : "icon-database"
+    },
+    "name": "zookeeper"
+  },
+  "parameters": [
+    {
+      "name": "ZOOKEEPER_IMAGE",
+      "description": "The name of Zookeeper Docker image to use",
+      "value": "openshift/zookeeper-346-fedora20"
+    }
+  ],
+  "items":[
+    {
+      "apiVersion": "v1beta1",
```

Comment:
OpenShift didn't have all the v1beta3 objects in place when he started this.

----- Original Message -----
> > +      "description": "Example of replicated Zookeeper 3.4.6 server",
> > +      "tags": "database,zookeeper",
> > +      "iconClass" : "icon-database"
> > +    },
> > +    "name": "zookeeper"
> > +  },
> > +  "parameters": [
> > +    {
> > +      "name": "ZOOKEEPER_IMAGE",
> > +      "description": "The name of Zookeeper Docker image to use",
> > +      "value": "openshift/zookeeper-346-fedora20"
> > +    }
> > +  ],
> > +  "items":[
> > +    {
> > +      "apiVersion": "v1beta1",
> 
> is there a reason it's not all v1beta3?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/1965/files#r29428904